composer.json

```diff
@@ -17,7 +17,7 @@
     "require": {
         "php": "^7.3",
```

Comment:
That should be `^7.3 | ^8` otherwise, you cannot install zero on a system with php8 only 😉

Reply:
I'm not sure why I was thinking of updating this at the end. 🤦🏻